FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuthErrors.h

```diff
@@ -458,6 +458,10 @@ typedef NS_ERROR_ENUM(FIRAuthErrorDomain, FIRAuthErrorCode){
      */
     FIRAuthErrorCodeRecaptchaSDKNotLinked = 17208,
 
+    /** Indicates that the long in credentials is invalid.
+     */
+    FIRAuthErrorCodeInvalidLoginCredentials = 17213,
```

Comment:
Can we reuse https://firebase.google.com/docs/reference/ios/firebaseauth/api/reference/Enums/FIRAuthErrorCode#firautherrorcodeinvalidcredential and update the error message?

That way, we do not have to use up one more error code. The issue we saw with duplicate enums (in web SDK) won't apply here, right?

Reply:
I think the enum issue applies here as well as the underlying value of enums is an integer in Objc.

Reply:
By reusing, we will basically just expand the definition of "kFIRAuthErrorMessageInvalidCredential" to read - "The supplied auth credential is incorrect, malformed or has expired.". You won't need a second enum at all. WDYT?

Reply:
Done.